src/components/Goods.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Goods.scss';
+
+export class Goods extends React.Component {
+  state = {
+    goods: [...this.props.goods],
+    selectValue: '1',
+  }
+
+  reverseGoods = () => {
```

Comment:
```suggestion
  handleReverse = () => {
```
https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364